packages/browsers/src/launcher.ts

```diff
@@ -236,19 +263,24 @@ class Process {
         process.exit(130);
       case 'SIGTERM':
       case 'SIGHUP':
-        this.kill();
+        this.close();
         break;
     }
   };
 
-  close(): Promise<void> {
+  async close(): Promise<void> {
+    await this.#runHooks();
     if (this.#exited) {
       return Promise.resolve();
     }
     this.kill();
     return this.#browserProcessExiting;
   }
 
+  closed(): Promise<void> {
```

Comment:
The `close` and `closed` at too similar (Especially give they have the same signature), either better naming (suggestions: closed -> exited or close -> shutdown) or comment would be appreciated.